test/examples/C++/.gitignore

```diff
@@ -0,0 +1 @@
+example.wasm
```

Comment:
We have example.wasm in root ignore list

Reply:
You mean we could have it there? For sure, but I suggest doing it explicitly in the language-specific subfolders.

Reply:
In C++ and AssemlyScript cases - I thing we can even have `*.wasm` in their ignore files to cover the cases where they produce a bin with a different name with, say:
```c++
em++ -std=c++11 example.cpp  -Os -s WASM=1 -s SIDE_MODULE=1 -o someNameForMyPurposes.wasm
```